{
  "uid" : "1e2becdb884840a1",
  "name" : "Create new user",
  "fullName" : "tests.unit_tests.test_users.TestUsers#test_create_user",
  "historyId" : "0f9fd8d4ef97e891848b2e0476d1c2f0",
  "time" : {
    "start" : 1740672120265,
    "stop" : 1740672120265,
    "duration" : 0
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: Init failed\nassert 401 == 205\n +  where 401 = <Response [401]>.status_code",
  "statusTrace" : "@pytest.fixture(autouse=True, scope=\"session\")\n    def init_environment():\n        response = requests.post(\n            url=f\"{HOST}/setup\",\n            headers={\n                \"Authorization\": f\"Bearer {os.getenv('API_TOKEN')}\"\n            }\n        )\n>       assert response.status_code == 205, \"Init failed\"\nE       AssertionError: Init failed\nE       assert 401 == 205\nE        +  where 401 = <Response [401]>.status_code\n\nconftest.py:27: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "init_environment",
    "time" : {
      "start" : 1740672120359,
      "stop" : 1740672123153,
      "duration" : 2794
    },
    "status" : "failed",
    "statusMessage" : "AssertionError: Init failed\nassert 401 == 205\n +  where 401 = <Response [401]>.status_code\n",
    "statusTrace" : "  File \"/usr/local/lib/python3.10/site-packages/pluggy/_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.10/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/usr/workspace/conftest.py\", line 27, in init_environment\n    assert response.status_code == 205, \"Init failed\"\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "stepsCount" : 0,
    "hasContent" : true
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1740672120266,
      "stop" : 1740672120359,
      "duration" : 93
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "stepsCount" : 0,
    "hasContent" : false
  } ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Administration"
  }, {
    "name" : "feature",
    "value" : "Users"
  }, {
    "name" : "tag",
    "value" : "users"
  }, {
    "name" : "parentSuite",
    "value" : "tests.unit_tests"
  }, {
    "name" : "suite",
    "value" : "test_users"
  }, {
    "name" : "subSuite",
    "value" : "TestUsers"
  }, {
    "name" : "host",
    "value" : "74cc032a4021"
  }, {
    "name" : "thread",
    "value" : "1-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.unit_tests.test_users"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ "users" ]
  },
  "source" : "1e2becdb884840a1.json",
  "parameterValues" : [ ]
}